packages/protobuf/src/messages.ts

```diff
@@ -1734,8 +1734,16 @@ export type ResetDevice = {
     backup_type?: Enum_BackupType;
 };
 
+export type Slip39Group = {
+    member_threshold: number;
+    member_count: number;
+};
+
 // BackupDevice
-export type BackupDevice = {};
+export type BackupDevice = {
+    group_threshold?: number;
+    groups: Slip39Group[];
```

Comment:
this is reauired, it it ok?

Reply:
hmm, if so, it would be probably better to pass empty array as a default? I am not sure how to do it in some sane way. Maybe @martykan got some idea? 🤔

Reply:
BackupDevice is a management restricted method, so I think it's not a big deal if there is a breaking change, since it shouldn't be used outside of Suite